server.go

```diff
@@ -285,6 +287,31 @@ func (s *Server) load() error {
 	})
 }
 
+func (s *Server) StartSelfMonitoring(database, retention string, interval time.Duration) error {
+	if interval == 0 {
+		return fmt.Errorf("statistics check interval must be non-zero")
+	}
+
+	go func() {
+		for {
+			time.Sleep(interval)
```

Comment:
Ah, yes, I know about the tickers. I agree it would be nice to be precise with this stuff.